book.rb

```diff
@@ -0,0 +1,11 @@
+class Book < Item
+  attr_accessor :publisher, :cover_be_archived, :author, publish_date, :label
+
+  def initialize(publisher, cover_be_archived, author, publish_date, label, id)
+    super(publish_date, author, label, id)
+    @publisher = publisher
+    @cover_be_archived = cover_be_archived
```

Comment:
- [ ] Kindly remove author and label from initialization as the requirement said "Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created."